scripts/publish.js

```diff
@@ -35,6 +35,7 @@ var tagName = [moduleName, newVersion].join('-');
 
 if (moduleName === 'google-cloud') {
   tagName = ('v' + newVersion);
+  cp(path.join(__dirname, '../README.md'), cwd);
```

Comment:
Is there a point to have a README in <kbd>packages/google-cloud</kbd> if we're just going to copy the root README?